src/ai/ai_gui.cpp

```diff
@@ -240,33 +240,33 @@ struct AIConfigWindow : public Window {
 			case WID_AIC_INCREASE_INTERVAL: {
 				int new_value;
 				if (widget == WID_AIC_DECREASE_INTERVAL) {
-					new_value = std::max(static_cast<int>(MIN_COMPETITORS_INTERVAL), GetGameSettings().difficulty.competitors_interval - 1);
+					new_value = std::max<int>(MIN_COMPETITORS_INTERVAL, GetGameSettings().difficulty.competitors_interval - 1);
 				} else {
-					new_value = std::min(static_cast<int>(MAX_COMPETITORS_INTERVAL), GetGameSettings().difficulty.competitors_interval + 1);
+					new_value = std::min<int>(MAX_COMPETITORS_INTERVAL, GetGameSettings().difficulty.competitors_interval + 1);
```

Comment:
remove these (and anywhere else they appear) - they are not purely codestyle and are potentially subtly altering the behaviour of the program